bin/loftsman-manifest-upload

```diff
@@ -80,11 +82,15 @@ process(){
                 fi
                 rm -f $name
             fi
-            if ! cray artifacts create config-data argo/loftsman/$PRODUCT_NAME/$PRODUCT_VERSION/manifests/"$(basename $manifest)" $manifest; then
-                echo "ERROR failed to create s3://config-data/argo/loftsman/$PRODUCT_NAME/$PRODUCT_VERSION/manifests/$(basename $manifest)"
+            manifest_name="$(basename $manifest)"
+            s3_manifest_key="argo/loftsman/$PRODUCT_NAME/$PRODUCT_VERSION/manifests/$manifest_name"
+            s3_manifest_path="config-data/$s3_anifest_key"
```

Comment:
I assume s3_anifest should be s3_manifest here...

Reply:
Fixed.